### ISSUE TITLE
Subsequent filter options to only display available options based on first filter selected

### ISSUE THREAD

**Issue by jbatterman** [opendcs/opendcs#735]
**Is your feature request related to a problem? Please describe.**
Having the GUI Filters is really nice, but are still difficult to navigate when selecting multiple filters at once.  For example, if a filter is selected for Agency, opening the filter list for Config still displays every config available in the database rather than just those that are currently displayed in the table based on the first filter.

**Describe the solution you'd like**
Would like subsequent filters to automatically refresh lists along with the table below, so that clicking a subsequent filter dropdown will display only those filters that are available in the table.

**Additional context**
After choosing a Platform filter, the options for a subsequent Config filter (in this case) should be limited to two, but instead displays every option:
![image](https://github.com/user-attachments/assets/4480f6b1-86dc-4615-988d-f56c72c67266)

**Comment by MikeNeilson** [comment]
This is a great idea, and not terrible difficult, but will be a bit time consuming to get right so punting to later release.

**Comment by jbatterman** [comment]
I agree with this - no longer applicable with the new filtering capabilities.
